src/status_im2/contexts/chat/messages/list/view.cljs

```diff
@@ -25,6 +25,7 @@
 (defonce ^:const loading-indicator-page-loading-height 100)
```

Comment:
The value of `:scroll-event-throttle` in this file needs to be `8` to have to 120 hz animations

Reply:
I partially agree with the suggestion @OmarBasem because I think we should probably be careful with this setting and not blindly use 8. Ideally, we should have a way to know if the hardware can take advantage of high refresh rates, otherwise we just make the experience worse for a bunch of users.

Obviously it's not something for this PR to implement, so I'll leave an issue open about this concern/problem in case somebody can work on it.

Reply:
@ilmotta there is an open issue :) https://github.com/status-im/status-mobile/issues/14924

Reply:
@ilmotta @OmarBasem 
I think there won't be a problem if we set the `scroll-event-throttle` to 8, since at much it is the screen's refresh rate.
But keep in mind this will only work on iOS, idk how to set it on Android.

please check: https://reactnative.dev/docs/0.69/scrollview#scrolleventthrottle-ios

Reply:
I agree with you @ulisesmac. We just need to be mindful IMO and not use 8 unless a significant difference can be felt in the UX, so by default start from higher values and lower only after proper experimentation. In some views doubling or tripling the frequency of events can be significant, because usually each call to the scroll event can cause a mutation in a ratom, which in turn re-renders the world.

Reply:
I agree, completely makes sense :+1: